client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,351 @@
+import os
+import sys
+import time
+import stat
+import shutil
+
+import king_phisher.client.plugins as plugins
+
+from gi.repository import Gtk
+from gi.repository.GdkPixbuf import Pixbuf
+
+import boltons.strutils
+import boltons.timeutils
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Josh Jacob']
+	title = 'SFTP Client GUI'
+	description = """
+	How many foos could fu foo is fu fooed fu foos on foo.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher'
+
+	def initialize(self):
+		module = sys.modules['king_phisher.client.application']
+		self.temp = module.KingPhisherClientApplication.do_sftp_client_start
+		module.KingPhisherClientApplication.do_sftp_client_start = self.new_SFTP_signal
+		sys.modules['king_phisher.client.application'] = module
+		return True
+
+	def _cleanup(self):
```

Comment:
Now, I'm assuming this is related to how the signal was connected. There shouldn't need to be a clean up method here yet. Eventually the cleanup method should stop and running transfers, cancel all pending transfers and close the window.